.github/workflows/ci.yml

```diff
@@ -24,24 +25,17 @@ jobs:
           - '3.9'
           - '3.10'
 
-    defaults:
-      run:
-        shell: bash -l {0}
-
     steps:
       - uses: actions/checkout@v3
-
-      - uses: conda-incubator/setup-miniconda@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
-          activate-environment: test
-
-      - run: conda install numpy scipy biopython pandas matplotlib pytest
-
-      - run: python setup.py install
-
-      - run: conda info
-
-      - run: conda list
-
-      - run: bash test.sh
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pytest wheel
+      - name: Install treetime
+        run: python -m pip install .
```

Comment:
_(non-blocking)_

In Augur, the dev dependencies are [defined in setup.py](https://github.com/nextstrain/augur/blob/67027a6ae6a7a324c8eb646593cc5c0a89650ccf/setup.py#L70-L87) and installed with the package using `pip install ".[dev]"`. I prefer that pattern but this is fine too!